javascript/stimulus_reflex.js

```diff
@@ -42,12 +46,13 @@ const resetImplicitReflexPermanent = event => {
 // controller - the StimulusReflex controller to subscribe
 //
 const createSubscription = controller => {
+  actionCableConsumer = actionCableConsumer || getConsumer()
   const { channel, room } = controller.StimulusReflex
-  const id = `${channel}${room}`
+  const identifier = JSON.stringify({ channel, room })
```

Comment:
We should do up a PR to remove room from the client and server. Nobody has mentioned it except as something that was confusing since we removed it from the docs.